artifact.md

```diff
@@ -0,0 +1,84 @@
+# OCI Artifact Manifest Specification
+
+The goal of the Artifact Manifest Specification is to define content addressable artifacts in order to store them along side container images in a registry. Like [OCI Images](manifest.md), OCI Artifacts may be referenced by the hash of their manifest. Unlike OCI Images, OCI Artifacts are not meant to be used by any container runtime.
+
+Examples of artifacts that may be stored along with container images are Software Bill of Materials (SBOM), Digital Signatures, Provenance data, Supply Chain Attestations, scan results, and Helm charts.
+
+This section defines the `application/vnd.oci.artifact.manifest.v1+json` [media type](media-types.md).
+For the media type(s) that this is compatible with see the [matrix](media-types.md#compatibility-matrix).
+
+# Artifact Manifest
+
+## *Artifact Manifest* Property Descriptions
+
+- **`mediaType`** *string*
+
+  This property MUST be used and contain the media type `application/vnd.oci.artifact.manifest.v1+json`.
+
+- **`artifactType`** *string*
+
+  This property SHOULD specify the mediaType of the referenced content and MAY be registered with [IANA][iana].
+
+- **`blobs`** *string*
+
+  This OPTIONAL property contains a list of [descriptors](descriptor.md).
+  Each descriptor represents an artifact of any IANA mediaType.
```

Comment:
I have a use case where each element in the `blobs` array is associated with the corresponding element in the `layers` array of the referent OCI image. In other words, the length of the two arrays are always equal. However, a blob in the referring artifact may be omitted for any particular image layer.

This can be addressed by adding to this line: "Any element in the list may also be `null`."

Posting here for comments. I can open a new PR if that's easier.

Reply:
I suspect a null descriptor entry may cause issues for tools. What if an annotation in the descriptor was used to indicate which layer it is referencing. That could be a layer digest, or just a list index number.

Reply:
This PR is defining the spec that the tools will have to conform to, no? But okay; I can go with the annotation approach. Thanks.